test/unit/helpers/jobs-queue.js

```diff
@@ -10,6 +10,7 @@ var rewire = require('rewire');
 var jobsQueue = require('../../../helpers/jobsQueue.js');
 var peers = rewire('../../../modules/peers');
 
+//These tests are breaking other tests (relying on setTimeout) running on the same process because of a time stubbing
```

Comment:
Are these tests failing now?

Reply:
These test will make other tests behaving unreliable - `clock.restore()` in after phase doesn't seem to be working. I left it as a warning.

As `test-unit` task now runs tests in separate processes it is not the issue.